pyccl/__init__.py

```diff
@@ -34,9 +34,6 @@
 # BCM stuff
 from .bcm import bcm_model_fka
 
-# Halo mass function
-from .massfunction import massfunc, massfunc_m2r, sigmaM, halo_bias
-
```

Comment:
Again more API breaks. Just an FYI. I know I suggested it.

Reply:
Yes, this is what I meant. This is not how we will compute mass functions anymore (sigmaM is still the same though)